src/Microsoft.ML/Runtime/EntryPoints/ImportTextData.cs

```diff
@@ -27,13 +27,23 @@ public sealed class Input
             public string CustomSchema = null;
         }
 
+        [TlcModule.EntryPointKind(typeof(ILearningPipelineLoader))]
+        public sealed class LoaderInput
+        {
+            [Argument(ArgumentType.Required, ShortName = "data", HelpText = "Location of the input file", SortOrder = 1)]
+            public IFileHandle InputFile;
+
+            [Argument(ArgumentType.Required, ShortName = "args", HelpText = "Arguments", SortOrder = 1)]
+            public TextLoader.Arguments Arguments = new TextLoader.Arguments();
+        }
+
         public sealed class Output
         {
             [TlcModule.Output(Desc = "The resulting data view", SortOrder = 1)]
             public IDataView Data;
         }
 
-        [TlcModule.EntryPoint(Name = "Data.TextLoader", Desc = "Import a dataset from a text file")]
+        [TlcModule.EntryPoint(Name = "Data.CustomTextLoader", Desc = "Import a dataset from a text file")]
         public static Output ImportText(IHostEnvironment env, Input input)
```

Comment:
We are keeping this around I suppose, but I wonder if we shouldn't have some mechanism for its eventual deprecation, since I imagine we don't plan to include it forever.

Typically I think this is accomplished using a `System.ObsoleteAttribute`. I wonder if the code-gen step could (1) check to see if this attribute is marked with that attribute and if so, migrate the same attribute over the autogenerated class or method (what is currently in `CSharpApi.cs`). I see you have changed `CSharpApiGenerator.cs`... that is, in the attribute list, if there is an `ObsoleteAttribute` with message `"foo"`, declare `[Obsolete("foo")]` on the generated class.

It strikes me as a good time to maybe add that... should I hope be a few lines of code, unless I'm mistaken, since you could get the obsolete attribute, if any, from `EntryPointInfo.Method.Attributes`.